Training/Image_Classifier.py

```diff
@@ -17,7 +17,7 @@
 
 # Load configuration file and name
 #config = toml.load(sys.argv[1])'"
-config = toml.load('./config_files/trainer_tumour_convnet.ini')
+config = toml.load('./config_files/preprocessing/trainer_tumour_convnet.ini')
```

Comment:
Please only keep sys.argv here otherwise it becomes unuusable

Reply:
Added a second commit to solve this.